recipe/bld_fftw.bat

```diff
@@ -9,7 +9,7 @@ cmake -LAH -G"NMake Makefiles"                             ^
   -DCMAKE_BUILD_TYPE="%CMAKE_CONFIG%"                      ^
   -DENABLE_THREADS=ON                                      ^
   -DWITH_COMBINED_THREADS=ON                               ^
-  -DENABLE_SSE2=ON                                         ^
+  -DHAVE_SSE2=ON                                           ^
```

Comment:
I think this will work, although it is technically a hack and may fail later. Please add a comment to make sure we don't forget, something like: `::We use HAVE_SSE2 instead of ENABLE_SSE2 because FFTW 3.3.10's CMakeLists.txt does not correctly enable SSE2 on 64-bit Windows; see https://github.com/FFTW/fftw3/issues/292.`. I will also create an issue in this repo for this.

Reply:
Done!